### PR TITLE
ERS-461 renaming table and columns

### PR DESCRIPTION
**Pull request by psjobergprivat**

### CODE REVIEW

File: application/src/main/java/eu/europa/ec/fisheries/uvms/activity/service/mapper/BaseMapper.java [hunk 1]
```diff
@@ -399,19 +399,19 @@ protected FLUXReportDocument mapToFluxReportDocument(FaReportDocumentEntity faRe
...
 
-        String fluxReportDocument_referenceId = faReportDocumentEntity.getFluxReportDocument_ReferenceId();
-        String fluxReportDocument_referenceIdSchemeId = faReportDocumentEntity.getFluxReportDocument_ReferenceIdSchemeId();
+        String fluxReportDocument_referencedFaReportDocumentId = faReportDocumentEntity.getFluxReportDocument_ReferencedFaReportDocumentId();
+        String fluxReportDocument_referencedFaReportDocumentSchemeId = faReportDocumentEntity.getFluxReportDocument_ReferencedFaReportDocumentSchemeId();
 
-        IDType referenceId = new IDType();
-        referenceId.setValue(fluxReportDocument_referenceId);
-        referenceId.setSchemeID(fluxReportDocument_referenceIdSchemeId);
+        IDType referencedId = new IDType();
```
Comment (RobinMilding) [inline application/src/main/java/eu/europa/ec/fisheries/uvms/activity/service/mapper/BaseMapper.java:405]: A subtle difference, but oh what a great change in meaning!

File: application/src/test/java/eu/europa/ec/fisheries/uvms/activity/service/util/MapperUtil.java [hunk 2]
```diff
@@ -736,7 +736,7 @@ private static MeasureType getMeasureType(int value, String unitCode, String lis
         faCatch_1[1] = "BEAGLE";
         faCatch_2[1] = "SEAFOOD";
         faCatch_3[1] = "BEAGLE";
-        faCatch_4[0] = "BEAGLE";
+        faCatch_4[1] = "BEAGLE";
```
Comment (RobinMilding) [inline application/src/test/java/eu/europa/ec/fisheries/uvms/activity/service/util/MapperUtil.java:739]: It disturbs me that this doesn't make a difference anywhere else

File: application/src/test/java/eu/europa/ec/fisheries/uvms/activity/service/util/MapperUtil.java [hunk 3]
```diff
@@ -518,7 +518,7 @@ public static FLUXLocation getFluxLocation() {
         return fluxLocation;
     }
 
-    public static FLUXReportDocument getFluxReportDocument() {
+    public static FLUXReportDocument getFluxReportDocumentLowerCase() {
```
Comment (RobinMilding) [inline application/src/test/java/eu/europa/ec/fisheries/uvms/activity/service/util/MapperUtil.java]: Is it possible to get even better method names? These are not identical FLUXReportDocuments, this one is a correction (I think, CodeType "5") and the other is a new report (CodeType "9").

**Review by RobinMilding — APPROVED** [review]
Change suggestion is optional, good job

File: application/src/test/java/eu/europa/ec/fisheries/uvms/activity/service/util/MapperUtil.java (hunk 2 above)
Comment (psjobergprivat) [reply application/src/test/java/eu/europa/ec/fisheries/uvms/activity/service/util/MapperUtil.java:739]: I agree, not very thorough tests...

File: application/src/test/java/eu/europa/ec/fisheries/uvms/activity/service/util/MapperUtil.java (hunk 3 above)
Comment (psjobergprivat) [reply application/src/test/java/eu/europa/ec/fisheries/uvms/activity/service/util/MapperUtil.java]: Good idea! I just saw that the cases of the Id was different. New/Correction is much better

**Review by RobinMilding — APPROVED** [review]

**Pull request merged**